lib/tasks_private/azure.rake

```diff
@@ -0,0 +1,24 @@
+namespace :spec do
+  namespace :regenerate do
+    base_dir = File.join(ManageIQ::Providers::Azure::Engine.config.root.to_s, 'spec')
+    cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/azure')
+    spec_dir = File.join(base_dir, 'models/manageiq/providers/azure')
+
+    desc "Regenerate all the cassettes"
+    task :all do
+      Dir["#{cass_dir}/**/*.yml"].each do |file|
+        FileUtils.rm(file, :verbose => true)
+      end
+      sh "bundle exec rspec"
```

Comment:
This is a weird way to call another take task...

Just a move so good with it but would be a good candidate to fix after